pyo3-macros-backend/src/module.rs

```diff
@@ -207,10 +207,12 @@ pub fn pymodule_module_impl(mut module: syn::ItemMod) -> Result<TokenStream> {
             }
             Item::Mod(item_mod) => {
                 ensure_spanned!(
-                    !has_attribute(&item_mod.attrs, "pymodule_export"),
+                    !has_attribute(&item_mod.attrs, &["pymodule_export"]),
                     item.span() => "`#[pymodule_export]` may only be used on `use` statements"
                 );
-                if has_attribute(&item_mod.attrs, "pymodule") {
+                if has_attribute(&item_mod.attrs, &["pymodule"])
+                    || has_attribute(&item_mod.attrs, &["pyo3", "prelude", "pymodule"])
```

Comment:
We should probably use `ctx.pyo3_path` instead of `pyo3` hard-coded here.

`pyo3::pymodule` is also a valid path to the macro, so we should allow that.

Reply:
Hmm, moved too fast, it's not clear what the best way to get an `&str` out of the `syn::Path` is. This may need more involved surgery.